scripts/train_unet.py

```diff
@@ -168,7 +165,7 @@ def score_function(engine):
     volumes_transformations = get_volumes_transformations(args.input_size, args.device)
     masks_transformations = get_masks_transformations(args.input_size, args.device)
 
-    train_set, valid_set = get_sets(args.volumes_path, args.masks_path, volumes_transformations, masks_transformations)
+    train_set, valid_set = get_sets(args.dataset_json, args.division_json, volumes_transformations, masks_transformations)
```

Comment:
Could this `dataset_json` file be also provided in the commit? Or at least some kind of structure, because it's unclear to me what exactly it should contain

Reply:
This json file is in dataset directory, provided by the hosts of the competition